parlai/crowdsourcing/tasks/multi_model_chat/task_config/onboard_task_data.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
do we generally open-source our onboarding ?

Reply:
We had it at least for the Wizard of Internet as far as I remember.